src/openms/include/OpenMS/ANALYSIS/ID/AccurateMassSearchEngine.h

```diff
@@ -52,7 +52,8 @@
 #include <vector>
 
 namespace OpenMS
-{
+{ 
+  typedef unsigned int uint;
```

Comment:
looking at the commits more closely, you probably did not intend to include
26868d9822d333383b211e1778e664cb5a7
please remove.